Lib/importlib/metadata.py

```diff
@@ -0,0 +1,271 @@
+import os
+import re
+import abc
+import sys
+import email
```

Comment:
Imports not alphabetized.

Reply:
These issues have been corrected upstream.